cumulusci/robotframework/Salesforce.py

```diff
@@ -160,6 +160,14 @@ def get_current_record_id(self):
                 return oid_match.group(2)
         raise AssertionError("Could not parse record id from url: {}".format(url))
 
+    def get_field_value(self, label):
+        """Return the current value of a form field based on the field label"""
+        input_element_id = self.selenium.get_element_attribute(
+            "xpath://label[contains(., '{}')]".format(label), "for"
```

Comment:
I suspect someone will ask "why isn't this xpath in locators.py"? I spoke briefly about this with Jason a few days ago. My personal position is that since this xpath is used in exactly one spot, it makes the code easier to understand if the xpath is hard-coded here rather than added to the locators data structure. Many of our locators are single-purpose, so making them sharable doesn't buy us anything but extra overhead.

Reply:
The point of putting the locators in a separate mapping is so that we can swap between mappings based on the Salesforce version if the DOM structure changes.

In this case, it seems pretty unlikely that the platform will stop using the `label` element to label fields. On the other hand, I could imagine the structure changing to add a div inside the label or something like that, which would mean needing to swap between two versions of the locator during the period when that release rolls out. On the third hand, we need to make updates to the package anyway to support that swapping and can move the locator to a mapping at that point if necessary. So I don't have a problem with this.